src/app.ts

```diff
@@ -1,21 +1,75 @@
+import * as dotenv from "dotenv"
+dotenv.config()
+
+import * as admin from "firebase-admin"
+import { SecretManagerServiceClient } from "@google-cloud/secret-manager"
+import { CoreConfig } from "./core/types_core"
 import * as telegram from "./telegram/bot_telegram"
 import * as discord from "./discord/bot_discord"
 import { TelegramConfig } from "./telegram/types_telegram"
 import { DiscordConfig } from "./discord/types_discord"
+import { Core } from "./core/core"
+import { Translator } from "./core/translator"
 
-const telegramRunning = process.argv.includes("--telegram")
-const discordRunning = process.argv.includes("--discord")
+export const config = new CoreConfig()
 
-if (telegramRunning) {
-  const telegramBot = new telegram.EnforcingTelegramBot(new TelegramConfig())
-  console.log("Started Telegram bot.")
-}
+admin.initializeApp()
 
-if (discordRunning) {
-  const discordBot = new discord.EnforcingDiscordBot(new DiscordConfig())
-  console.log("Started Discord bot.")
-}
+const TELEGRAM_TOKEN_NAME = "projects/telegram-lang-enforcer/secrets/TELEGRAM_TOKEN/versions/latest"
+const TELEGRAM_TOKEN_NAME_DEV = "projects/telegram-lang-enforcer/secrets/TELEGRAM_TOKEN_DEV/versions/latest"
+const DISCORD_TOKEN_NAME = "projects/telegram-lang-enforcer/secrets/DISCORD_TOKEN/versions/latest"
```

Comment:
This is a bit hardcoded

Reply:
they we'll tell people to name their projects with this name in the GCP console.

BTW I'm fed up with this GCP secret manager bullshit and will likely remove it from `app.ts` and leave good, old, plain env vars

Reply:
Ah, a sigh of relief